### DISCUSSION TITLE
運用class出問題

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#46] [Q&A] [answered]
按照教學內容輸入，但是他反應說在那個類別裡沒有show_question
想問一下是哪裡出了問題，感謝！
**（第一張是類別）**
![3331](https://user-images.githubusercontent.com/90883526/136678217-d4908f33-5a63-49c7-bf33-5f5cddd9a23c.png)
**（第二張是我想拿來用的程式）**
![3357](https://user-images.githubusercontent.com/90883526/136678218-a4470a87-f6d2-4f2c-837b-428075aa3060.png)

**Comment by jacky0115** [comment, marked as answer]
你有試著restart kernel看看嗎?

**Reply by bonny123865** [reply]
阿呀，剛剛重新再試一次就成功了!
感覺很不好意思，但還是謝謝你的提醒(°ー°〃)
